plugins/outputs/mongodb/mongodb.go

```diff
@@ -0,0 +1,189 @@
+package mongodb
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+type MongoDB struct {
+	serializer serializers.Serializer
+
+	URI             string   `toml:"uri"`
+	DB              string   `toml:"db"`
+	Collection      string   `toml:"collection"`
+	Timeout         uint64   `toml:"timeout"`
```

Comment:
You might want to use
```suggestion
	Timeout         internal.Duration   `toml:"timeout"`
```
here to allow the specification of timeout = "2m" in the config.